cmd/bridge/main.go

```diff
@@ -148,12 +148,13 @@ func main() {
 	switch branding {
 	case "okd":
 	case "okdvirt":
+	case "openshiftvirt":
```

Comment:
I've added the `okdvirt` on previous line for "out" customization.

I think you just need to `update` the already existing `.svg` file.